data/transactions/logic/assembler_test.go

```diff
@@ -527,7 +527,7 @@ func assemblyTrace(text string, ver uint64) string {
 
 func testProg(t testing.TB, source string, ver uint64, expected ...Expect) *OpStream {
 	t.Helper()
-	program := strings.ReplaceAll(source, ";", "\n")
+	program := source
```

Comment:
I suppose we should just use `source`

Reply:
I'm not sure if this is an old commit or I messed up the merge, but try refreshing and see if program is still used.

Reply:
Nope, all set.